ember-lottie/src/components/lottie.ts

```diff
@@ -113,10 +113,15 @@ export default class LottieComponent extends Component<LottieSignature> {
     const speed = Ember.testing ? 0 : this.args.speed || 1;
     this.animation.setSpeed(speed);
 
-    this.mediaQuery?.addEventListener(
-      'change',
-      this.handleReducedMotionPreferenceChange,
-    );
+    if (this.mediaQuery?.addEventListener) {
+      this.mediaQuery?.addEventListener(
```

Comment:
nitpick: don't need to check mediaQuery existence here if it's condition above is true

Reply:
Yes, makes sense! I pushed the change.

For the `else` block, though, I would still need the optional chaining. Would you turn that into a `else if (this.mediaQuery?.addListener)` and remove the optional chaining in that block as well?

Reply:
Either, no strong opinion.

Reply:
I refactored to `else if`. In the end, there's no benefit in having catch-all `else` statement and in this way it's a little bit more descriptive. Thank you for your review!